visualization/app/codeCharta/ui/viewCube/viewCube.mouseEvents.service.ts

```diff
@@ -62,9 +62,12 @@ export class ViewCubeMouseEventsService {
 	}
 
 	private transformIntoCanvasVector(event: MouseEvent) {
```

Comment:
This seems to be very similar to the function in visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

Maybe we can reuse the logic.